.azure-pipelines/default.yml

```diff
@@ -0,0 +1,40 @@
+trigger:
+- master
+
+jobs:
+- job: 'Windows'
+
+  pool:
+    vmImage: 'vs2017-win2016'  
+
+  steps:
+  - script: |
+     python -m venv .\
+     
+     pip install git+https://github.com/lmazuel/swagger-to-sdk
+     pip install -e "git+https://github.com/Azure/azure-sdk-for-python#subdirectory=azure-sdk-tools&egg=azure-sdk-tools"
+
+    displayName: 'Prep Environment, Install Packages copy'
+
+  - powershell: |
+     $inputFilterSet="$(build_targeting_string)"
+     
+     if ($inputFilterSet -eq "")
+     {
+         $inputFilterSet = "azure-*"
+     }
+     
+     foreach($inputFilter in "$inputFilterSet".split(","))
+     {
+       $foldersToIterate=Get-ChildItem -Recurse -Filter "$inputFilter"
```

Comment:
nit: Minor but I personally like using the "dir" alias over Get-ChildItem. I would write this line as `$foldersToIterate = dir -r -i $inputFilter`